tasks/main.yml

```diff
@@ -25,3 +25,5 @@
   when: peering_manager_setup_systemd
 - include: setup_web_frontend.yml
   when: peering_manager_setup_web_frontend
+- include: setup_cron.yml
```

Comment:
Adding a condition for user to choose cron setup could be a plus. For instance, we could define a `peering_manager_setup_cron_jobs: true` in `defaults/main.yml` and add `when: peering_manager_setup_cron_jobs` here.